src/current/v23.2/sso-db-console.md

```diff
@@ -6,94 +6,104 @@ docs_area: manage
 key: sso.html
 ---
 
+CockroachDB clusters allow users to authenticate with Single sign-on (SSO), both to the [DB Console]({% link {{ page.version.version }}/ui-overview.md %}), and for SQL client access, using OpenID connect (OIDC).
```

Comment:
Same note about OIDC

Reply:
Ok, I think I've got this broken out correctly now, please confirm.

Reply:
Same note I left above about still seeing a reference to OIDC and cluster sso